aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -401,6 +417,23 @@ public class AwsProperties implements Serializable {
    */
   public static final String LAKE_FORMATION_DB_NAME = "lakeformation.db-name";
 
+  /**
+   * Used by {@link LakeFormationAwsClientFactory}. The maximum size of the lakeformation
+   * credentials cache, default to 100.
+   */
+  public static final String LAKE_FORMATION_MAX_CACHE_SIZE = "lakeformation.max-cache-size";
```

Comment:
nit: `cache.max-size` and `cache.expiry-millis`?

Reply:
actually, I notice we have a config `CatalogProperties.CACHE_EXPIRATION_INTERVAL_MS`, let's use the same name `lakeformation.cache.expiration-interval-ms`